apiserver/common/tools.go

```diff
@@ -24,7 +24,7 @@ import (
 	applicationerrors "github.com/juju/juju/domain/application/errors"
 	machineerrors "github.com/juju/juju/domain/machine/errors"
 	modelagenterrors "github.com/juju/juju/domain/modelagent/errors"
-	"github.com/juju/juju/environs/simplestreams"
+	stubservice "github.com/juju/juju/domain/stub"
```

Comment:
i think this import alias should be `stubdomain` or something similar. I don't think that service part comes into it. Service is a type within this package not the package.